packages/client-sdk-nodejs/src/auth-client.ts

```diff
@@ -0,0 +1,65 @@
+import {AuthClient as InternalAuthClient} from './internal/auth-client';
+import {IAuthClient} from '@gomomento/sdk-core/dist/src/internal/clients/auth/IAuthClient';
+import {AbstractAuthClient} from '@gomomento/sdk-core/dist/src/internal/clients/auth/AbstractAuthClient';
+import {MomentoLogger, RefreshApiToken} from '.';
+import {AuthClientProps} from './auth-client-props';
+import {
+  GenerateApiToken,
+  ExpiresIn,
+  CredentialProvider,
+} from '@gomomento/sdk-core';
+
+export class AuthClient extends AbstractAuthClient implements IAuthClient {
+  private readonly logger: MomentoLogger;
+
+  constructor(props: AuthClientProps) {
```

Comment:
I was thinking we could ditch this here in the public API for now, not just the internal one, sorry if that wasn't clear.